index.html

```diff
@@ -32,28 +32,12 @@
       <br />
 
       <p>
-        Lorem ipsum dolor sit amet consectetur adipisicing elit. Officia
-        corporis eius, fuga nam enim dolore vitae eligendi beatae ipsum eveniet
-        amet atque iure nobis reiciendis unde, deleniti neque mollitia, est
-        porro esse recusandae. Qui quaerat reiciendis accusamus incidunt, velit
-        doloremque aliquid nostrum modi, animi et praesentium eveniet, iure
-        temporibus tempore nesciunt. Molestias alias assumenda sint neque
-        doloremque dicta error nobis molestiae eveniet, quisquam, voluptates
-        cupiditate, amet ullam necessitatibus illum corrupti reiciendis quod
-        animi culpa expedita! Nam perferendis iusto repudiandae officiis beatae
-        praesentium? Sint et sapiente possimus explicabo at laudantium odit eos.
-        Accusamus ducimus rem explicabo doloribus magni omnis soluta, adipisci
-        nostrum mollitia enim dignissimos et laborum odit possimus harum iusto
-        repellat sit quibusdam facilis earum? Minus, alias numquam non facilis
-        dolorum ab sequi consequatur ratione minima, officiis ipsam, similique
-        harum eius omnis veritatis quod? Est saepe laboriosam voluptatibus esse
-        ut ea nam consectetur recusandae quo quis, aut temporibus praesentium
-        quam facere! Odit quas itaque nisi aliquid consequatur! Suscipit placeat
-        sint vitae sapiente obcaecati asperiores adipisci voluptatibus! Cumque
-        omnis sapiente tenetur nisi inventore quibusdam. Aliquid vero, iure
-        labore soluta minus fuga rem sapiente dolor voluptatem, pariatur rerum
-        saepe magni dolorum itaque sequi dignissimos quo officia aliquam nisi
-        eveniet. Consectetur, cupiditate sunt!
+        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Dicta quo eos
+        fugiat recusandae dolore magni, dolores quaerat nemo neque culpa iusto
+        nesciunt pariatur. Amet quaerat similique incidunt quam, harum magnam
+        soluta, temporibus, nobis est quis quia quisquam autem reprehenderit
+        iusto iure vero asperiores! Perferendis, recusandae necessitatibus
+        provident sunt assumenda harum.
       </p>
```

Comment:
You could use a br tag after this i think!

Reply:
Sure I can do that